<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #000000
</string>
  </property>
  <widget class="QLabel" name="YouSureLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #000000
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; color:#7011ff;&quot;&gt;You Died&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="KilledLine">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #666661; border-style: #000000;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="DiedLine">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>340</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #666661; border-style: #000000;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="PictureLabel">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>670</y>
     <width>821</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeadMenuButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>530</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #666661; border-style: #000000;</string>
   </property>
   <property name="text">
    <string>Main Menu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeadExitGameButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>530</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #666661; border-style: #000000;</string>
   </property>
   <property name="text">
    <string>Exit Game</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ScoreLine">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>440</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #666661; border-style: #000000;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
